.circleci/config.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       - run: ./ci/setup_ci_environment.sh
       - run: ./ci/install_bazelisk.sh
       - run: ./ci/install_gcc48.sh
-      - run: CC=/usr/bin/g++-4.8 ./ci/do_ci.sh bazel.test
+      - run: CC=/usr/bin/g++-4.8 ./ci/do_ci.sh bazel.test_api
```

Comment:
API is gcc 4.8 compatible but SDK is not?

Reply:
protobuf can't work with gcc-4.8.

Maybe I should exclude just some portions of the sdk from legacy compiler tests?

Reply:
Up to you. Sorry to hear about protobuf.